dataland-frontend/src/components/resources/dataTable/modals/GdvListOfBaseDataPointModal.vue

```diff
@@ -29,10 +29,10 @@ const tableData = computed(() => {
     <Column :header="dialogData.documentColumnHeader" headerStyle="width: 15vw;">
       <template #body="{ data }">
         <DocumentLink
+          :show-icon="!(data.dataSource == undefined)"
```

Comment:
Shouldn't this rather be a show if on the DocumentLink? I mean effectively it makes no difference.

Reply:
Yes I thought about it, but then I saw that DocumentLink already has a "show-icon" prop to control the display of the icon.
But I can also put the whole thing into a v-if if you want.   For me both is ok.
What do you think?

Reply:
I find show-if makes more sense but it doesn't really matter. I'll stop wasting our time and close this.